Add B2C app page (#10491)
MicrosoftDocs/msteams-docs: msteams-platform/get-started/b2c-apps.md

The image shows an example of the Teams interoperable meeting and calling experience you can embed into your website or native app:
![Alt text](../assets/images/call-with-chat-composite-hero.png "Screenshot of the meeting and calling experiences you can embed in your own app or website")

## Virtual Appointments

Virtual appointments are a communication pattern where a consumer and a business assemble for a scheduled appointment. The organizational boundary between consumer and business, and scheduled nature of the interaction, are key attributes of most virtual appointments. Many industries operate virtual appointments: meetings with a healthcare provider, a loan officer, or a product support technician.

The key technical steps for building a virtual appointments application:

1. Build a communication management service function using Graph Online Meeting APIs. This service schedules the meeting and configures options such as the availability of recording.
2. Embed Azure Communication Services Calling and Chat into your website or native app.
3. Program the communication management service to share the Teams meeting meta-data with the client application.

> [Azure Communication Services client libraries](../../../azure/articles/communication-services/concepts/sdk-options) are available for various platforms and languages, including Web browsers (JavaScript), iOS (Swift), Android (Java), Windows (.NET). The client libraries support both mobile and desktop web browsers. An open-source [UI library](../../../azure/communication-services/concepts/ui-library/ui-library-overview) can accelerate development for Web, iOS, and Android apps. Azure Communication Services is identity-agnostic, and you control how end users are identified and authenticated.

Reviewer: ```suggestion
[Azure Communication Services client libraries](../../../azure/articles/communication-services/concepts/sdk-options.md) are available for various platforms and languages, including Web browsers (JavaScript), iOS (Swift), Android (Java), Windows (.NET). The client libraries support both mobile and desktop web browsers. An open-source [UI library](../../../azure/communication-services/concepts/ui-library/ui-library-overview.md) can accelerate development for Web, iOS, and Android apps. Azure Communication Services is identity-agnostic, and you control how end users are identified and authenticated.
```

- **Scale.** Small businesses may have a few employees operating as agents in a limited role, for example a restaurant offering a phone number for reservations. While an airline may have thousands of employees and vendors providing a 24/7 contact center.
- **Channel.** Organizations can reach consumers through the phone system, apps, SMS, or consumer communication platforms such as WhatsApp.
- **Organizational approach.** Most businesses have employees operate as agents using Teams or a licensed contact center as a service software (CCaaS). Other businesses may outsource the agent role or use specialized service providers who fully operate contact centers as a service.

The key technical steps for building a virtual appointments application:

1. Plan and configure Teams auto attendants and call queues.
2. Build a communication management service function using Graph APIs to retrieve meta-data for the auto attendants and call queues.

> 3. Embed Azure Communication Services Calling and Chat into your website or native app.

Reviewer 2: Global: Request you to check the reference link added.

![Alt text](https://azure.github.io/communication-ui-library/images/call-with-chat-composite-hero.png "Screenshot of the meeting and calling experiences you can embed in your own app or website")

## Virtual Appointments

Virtual appointments are a communication pattern where a consumer and a business assemble for a scheduled appointment. The organizational boundary between consumer and business, and scheduled nature of the interaction, are key attributes of most virtual appointments. Many industries operate virtual appointments: meetings with a healthcare provider, a loan officer, or a product support technician.

The key technical steps for building a virtual appointments application:

1. Build a communication management service function using Graph Online Meeting APIs. This service schedules the meeting and configures options such as the availability of recording.
2. Embed Azure Communication Services Calling and Chat into your website or native app.
3. Program the communication management service to share the Teams meeting meta-data with the client application.

> [Azure Communication Services client libraries](https://learn.microsoft.com/azure/communication-services/concepts/sdk-options) are available for various platforms and languages, including Web browsers (JavaScript), iOS (Swift), Android (Java), Windows (.NET). Both mobile and desktop web browsers are supported. An open-source [UI library](https://learn.microsoft.com/azure/communication-services/concepts/ui-library/ui-library-overview) can accelerate development for Web, iOS, and Android apps. Azure Communication Services is identity agnostic and you control how end users are identified and authenticated.

Reviewer: ```suggestion
[Azure Communication Services client libraries](https://learn.microsoft.com/azure/communication-services/concepts/sdk-options) are available for various platforms and languages, including Web browsers (JavaScript), iOS (Swift), Android (Java), Windows (.NET). The client libraries support both mobile and desktop web browsers. An open-source [UI library](https://learn.microsoft.com/azure/communication-services/concepts/ui-library/ui-library-overview) can accelerate development for Web, iOS, and Android apps. Azure Communication Services is identity-agnostic, and you control how end users are identified and authenticated.
```

> [Azure Communication Services client libraries](https://learn.microsoft.com/azure/communication-services/concepts/sdk-options) are available for various platforms and languages, including Web browsers (JavaScript), iOS (Swift), Android (Java), Windows (.NET). The client libraries support both mobile and desktop web browsers. An open-source [UI library](https://learn.microsoft.com/azure/communication-services/concepts/ui-library/ui-library-overview) can accelerate development for Web, iOS, and Android apps. Azure Communication Services is identity-agnostic, and you control how end users are identified and authenticated.

Reviewer: ```suggestion
[Azure Communication Services client libraries](../../../azure/communication-services/concepts/sdk-options) are available for various platforms and languages, including Web browsers (JavaScript), iOS (Swift), Android (Java), Windows (.NET). The client libraries support both mobile and desktop web browsers. An open-source [UI library](../../../azure/communication-services/concepts/ui-library/ui-library-overview) can accelerate development for Web, iOS, and Android apps. Azure Communication Services is identity-agnostic, and you control how end users are identified and authenticated.
```

For more information and quickstarts, check out:

> - [Concept: Virtual visit apps with Azure Communication Services](https://learn.microsoft.com/azure/communication-services/tutorials/virtual-visits)

Reviewer: ```suggestion
- [Concept: Virtual visit apps with Azure Communication Services](../../../azure/communication-services/tutorials/virtual-visits.md)
```

> - [Concept: Azure & Teams interoperability](https://learn.microsoft.com/azure/communication-services/concepts/interop/guest/overview)

Reviewer: ```suggestion
- [Concept: Azure & Teams interoperability](/azure/communication-services/concepts/interop/guest/overview.md)
```

- [Azure Architecture Guide for joining a Teams meeting](https://learn.microsoft.com/azure/architecture/guide/mobile/azure-communication-services-architecture#microsoft-365-and-teams)
- [Azure Sample Builder for joining a Teams meeting](https://aka.ms/acs-sample-builder)
- [Quickstart: Join a Teams meeting as an external user](https://learn.microsoft.com/azure/communication-services/quickstarts/voice-video-calling/get-started-teams-interop?pivots=platform-android)

## Contact center

> Contact center applications are focused on unscheduled communication between consumers and agents. The unscheduled nature of the interaction is a key attribute of contact center applications. The term "contact center" captures a large family of applications diverse across scale, channels, and organizational approach:

Reviewer: ```suggestion
Contact center applications focus on unscheduled communication between consumers and agents. The unscheduled nature of the interaction is a key attribute of contact center applications. The term "contact center" captures a large family of applications diverse across scale, channels, and organizational approach:
```

- **Scale.** Small businesses may have a few employees operating as agents in a limited role, for example a restaurant offering a phone number for reservations. While an airline may have thousands of employees and vendors providing a 24/7 contact center.
- **Channel.** Organizations can reach consumers through the phone system, apps, SMS, or consumer communication platforms such as WhatsApp.

> - **Organizational approach.** Most businesses have employees operate as agents using Teams or a licensed contact center as a service software (CCaaS). Other businesses may out-source the agent role or use specialized service providers who fully operate contact centers as a service.

Reviewer: ```suggestion
- **Organizational approach.** Most businesses have employees operate as agents using Teams or a licensed contact center as a service software (CCaaS). Other businesses may outsource the agent role or use specialized service providers who fully operate contact centers as a service.
```

The key technical steps for building a virtual appointments application:

> 1. Plan and configure Teams autoattendants and call queues.

Reviewer: ```suggestion
1. Plan and configure Teams auto attendants and call queues.
```
